Transport/GpsSender.php

```diff
@@ -63,9 +64,13 @@ public function send(Envelope $envelope): Envelope
             $messageBuilder = $messageBuilder->setAttributes($attributesStamp->getAttributes());
         }
 
+        $senderOptionsStamp = $envelope->last(GpsSenderOptionsStamp::class);
+        if ($senderOptionsStamp instanceof GpsSenderOptionsStamp) {
+            $options = $senderOptionsStamp->getOptions();
+        }
         $this->pubSubClient
             ->topic($this->gpsConfiguration->getTopicName())
-            ->publish($messageBuilder->build())
+            ->publish($messageBuilder->build(), $options)
```

Comment:
`$options` might be undefined if `GpsSenderOptionsStamp` is not in the envelope

Reply:
yes, please put the `$options = []` before the if statement or put here `$options ?? []`